packages/manager/modules/bm-server-components/src/technical-details/technical-details.controller.js

```diff
@@ -92,4 +106,18 @@ export default class TechnicalDetailsController {
       this.technicalDetails.storage?.upgradable?.length
     );
   }
+
+  loadData() {
+    this.loading = true;
+    this.ovhFeatureFlipping
+      .checkFeatureAvailability('dedicated-server:upgradeWithTicket')
```

Comment:
I feel we can still pass in this feature availability from the place we are using this component. Because, the name of the feature might change, based on where this is being used....

Reply:
This will be used from two places. Dedicated server and Nutanix. The name is same in both the places. Remember this is to check if feature is available on dedicated server. I am sure that this will remain same irrespective of from where its been used.